recipes/libblitz/meta.yaml

```diff
@@ -0,0 +1,34 @@
+{% set version = "0.10" %}
+
+package:
+  name: libblitz
+  version: {{ version }}
+
+source:
+  fn: blitz-{{ version }}.tar.gz
+  url: http://downloads.sourceforge.net/project/blitz/blitz/Blitz%2B%2B%20{{ version }}/blitz-{{ version }}.tar.gz
+  md5: 66268b92bda923735f2e3afc87dcb58a
+
+build:
+  number: 0
+  skip: true   # [win]
+
+requirements:
+  build:
+    - autoconf
+    - automake
+    - libtool
+    - gcc
+
+test:
+  commands:
+    - test -d $PREFIX/include/blitz  # [unix]
+
+about:
+  home: https://sourceforge.net/projects/blitz
+  license: Artistic License 2.0, BSD, LGPLv3
+  summary: Blitz++ is a C++ class library for scientific computing which provides performance on par with Fortran 77/90
+
+extra:
+  recipe-maintainers:
+    - bioidiap
```

Comment:
> Personally, I think I would want to see individual people from that org so we aren't pinging the whole org.

Agreed. (I am think the members of that org would agree as well :wink:)

> Plus, I'm not sure how our logic behaves there. Then again I could see some pluses to this. Thoughts?

Even if the logic works I don't think that is advisable.